### ISSUE TITLE
Want to use InternImage for distillation of small models in cityscpae segmantation task!!!! but found that it is impossible to batch output 19 classes of logits ( where the did the output category changed to 19 types???). Thanks very much!!

### ISSUE THREAD

**Issue by duyunqi** [OpenGVLab/InternImage#149]

**Comment by OrangeSodahub** [comment]
。。。
